docs/resources/equinix_network_device.md

```diff
@@ -163,7 +234,9 @@ The `secondary_device` block supports the following arguments:
 * `hostname` - (Optional) Secondary device hostname.
 * `license_token` - (Optional) License Token can be provided for some device types o the device.
 * `license_file` - (Optional) Path to the license file that will be uploaded and applied on a
-secondary device. Applicable for some devices types in BYOL licensing mode.
+secondary device. Applicable for some device types in BYOL licensing mode.
+* `license_file_id` - (Optional) Identifier of a license file that will be applied on a secondary device.
```

Comment:
Should a customer be able to specify both `license_file` and `license_file_id`?  If not, we should call that out here in the docs.

Reply:
README docs have been updated to reflect that.